config/webpack/environment.js

```diff
@@ -1,3 +1,15 @@
 const { environment } = require('@rails/webpacker')
 
-module.exports = environment
+environment.plugins.append("Provide", new webpack.ProvidePlugin({
+
+    $: 'jquery',
+  
+    jQuery: 'jquery',
+  
+    Popper: ['popper.js', 'default']
```

Comment:
In particular, I do not recommend using jquery, since JS has the support to do what jquery years ago allowed to do.

Reply:
There is also already a npm package `react-popper`, so probably you can use the example shown here: https://popper.js.org/react-popper/v2/#example